CONTRIBUTORS

```diff
@@ -73,6 +74,7 @@ Thank you!
     Axel Westerhold <ml.awesterhold@dts.de>
     Aymeric Vincent <aymericvincent@free.fr>
     Barry Dobyns <barry@dobyns.com>
+    Ben Kallus <49924171+kenballus@users.noreply.github.com>
```

Comment:
Please avoid adding duplicate entries, especially duplicates with generated emails. We spend non-trivial amount of resources to maintain new CONTRIBUTORS in master. Our v7 branch should reuse that work.

```suggestion
```

Reply:
This was an automated addition by the same scripts being used on master.